StudentList/Repositories/StudentsRepository.cs

```diff
@@ -111,15 +110,20 @@ public StudentsRepository(Activity activity)
                 }
             }
 
-            await Task.Delay(1000);
+            await Task.Delay(this.delays.GetStudentsDelay);
 
             return temp.ToList();
         }
 
-        private ValidationResult Validate(string name, string birthdate, string group, string uni, string phone)
+        private ValidationResult Validate(SavingPhotoResult photo, string name, string birthdate, string group, string uni, string phone)
         {
             var validationResult = new ValidationResult();
 
+            if (photo.IsError)
+            {
+                validationResult.Errors.Add(nameof(photo), new List<string> { this.activity.GetString(Resource.String.photo_toast_error) });
```

Comment:
Don't use Context.GetString inside repository, it must be a pure .net class w/o refs on android.